### DISCUSSION TITLE
[Help]: Use Embla as a draw wheel

### DISCUSSION THREAD

**Discussion by peacepostman** [davidjerleke/embla-carousel#457] [Help]
Hello !

I am already using embla in our app and I was wondering if there is a way to achieve such effect with embla library and i would like to know if the lib can be used as a wheel to draw a winner, the idea is to have an array of nicknames displayed as this:

<img width="566" alt="idea" src="https://user-images.githubusercontent.com/16117861/233110775-01a113c5-7382-4029-b1fe-46f0d6a737cb.png">

We only show 5 entries, there is no possible interaction with the carousel and the wheel spin for x seconds then reach the winner index by reducing its speed progressively, a bit like the simulated effect used with `dragFree` . So each entry apply a translateY then disappear out of view.

I've studied the ios date picker demo and i think something can be achieved but i'm not sure about the initial fake drag gesture behaviour and its possible duration.

Thanks in advance for any answer,
Have a nice day

**Comment by davidjerleke** [comment]
Hi @peacepostman,

Thank you for your question. As much as I want to help everyone with their use cases, just maintaining the core, integrations, plugins and documentation is eating all my spare time budget, because I'm all alone in this project. 

I think what you want to do is possible with Embla, but I can't say for sure before giving it a try. I would suggest you to check out the `emblaApi.internalEngine()` method. Assuming that you're using the latest release, maybe:

- `emblaApi.internalEngine().scrollBody.useDuration(duration: number)`
- `emblaApi.internalEngine().scrollBody.useFriction(friction: number)` 
 
...could be of use in this particular case. It might be sufficient for you to use the tweening approach used on the [tween examples](https://www.embla-carousel.com/examples/static/#tween-examples) and tweak it a bit.

Best of luck!
David
